static/configs/service-constants.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Version": "1",
+  "UseLegacyDataPlaneSSO": true,
```

Comment:
What is this for? It's not really a constant like everything else?

Reply:
Compatiblity flag that we planning to use to turn off MAS-SSO login